### PR TITLE
[DropdownMenu] Make sure a disabled trigger doesn't open

### LINKED ISSUE #972: [DropdownMenu] Disabled trigger doesn't prevent menu from opening
This is because we are using `onPointerDown` instead of `onClick` and not checking `disabled`.

### PR DESCRIPTION
**Pull request by benoitgrelard**
Closes #972

### CODE REVIEW

File: packages/react/dropdown-menu/src/DropdownMenu.tsx [hunk 1]
```diff
@@ -164,20 +164,21 @@ const DropdownMenuTrigger = React.forwardRef<DropdownMenuTriggerElement, Dropdow
           aria-expanded={context.open ? true : undefined}
           aria-controls={context.open ? context.contentId : undefined}
           data-state={context.open ? 'open' : 'closed'}
+          disabled={disabled}
```
Comment (andy-hook) [inline packages/react/dropdown-menu/src/DropdownMenu.tsx:167]: Should we add the `data-disabled` attr as well?

**Review by andy-hook — COMMENTED** [review]
Do you remember the reason we went for `pointerDown` here while the popover trigger is `onClick`?

**Comment by benoitgrelard** [comment]
> Do you remember the reason we went for `pointerDown` here while the popover trigger is `onClick`?

Yes, because it enables quick actions in one pointer cycle, rather than 2. I can pointer down on the trigger, move, and pointer up on an item in one cycle. That's what MacOS does too.

File: packages/react/dropdown-menu/src/DropdownMenu.tsx (hunk 1 above)
Comment (benoitgrelard) [reply packages/react/dropdown-menu/src/DropdownMenu.tsx:167]: Good shout, let's add it for consistency.

**Review by andy-hook — APPROVED** [review]

**Pull request merged**